packages/core/src/components/Alert/Alert.jsx

```diff
@@ -49,6 +49,7 @@ Alert.propTypes = {
   ariaLabelledBy: PropTypes.string,
   children: PropTypes.node.isRequired,
   heading: PropTypes.string,
+  headingId: PropTypes.string,
```

Comment:
I think I fixed the props error with this

Reply:
Yep, that's what it wanted since it was being referenced in the constructor